src/www/ui/template/admin_license_file.js.twig

```diff
@@ -0,0 +1,118 @@
+{# Copyright 2017 Siemens AG
+
+   Copying and distribution of this file, with or without modification,
+   are permitted in any medium without royalty provided the copyright notice and this notice are preserved.
+   This file is offered as-is, without any warranty.
+#}
+
+var result = [];
+var uri = "{{ tracebackURI }}";
+
+$(document).ready(function () {
+    createLicenseTable();
+
+    //
+    //  Set up asynchronous loading of complete license texts
+    //
+
+    // Note: At this point one has to use delegated event handlers.
+    //  Ordinaray handlers are only registered for elements that exist in the DOM at the time of registration
+    //  Datatable pagination breaks this as ordinary handlers are only registered for elements of the one visible table page.
+    $('#adminLicenseTable').on('mouseover', '.licenseTextArea', function() {
+        if( $(this).data('textloaded')==false ) {
+            setLicenseText($(this).data('licenseid'), $(this).attr('id'))
+            $(this).data('textloaded',true);
+        }
+
+    });
+});
+
+function setLicenseText(licenseID, textareaID) {
+    var result = $.ajax({
+        type: "GET",
+        data: {getLicenseText : true, licenseID: licenseID},
+        url: "?mod=admin_license",
+        success: function (data) {
+            $('#'+textareaID).val(data);
+        },
+        error: function () {
+            $('#'+textareaID).val("Error loading the license text");
+        }
+    });
+}
+
+//
+//  Set up License table
+//
+
+{% for entry in data %}
+result.push({
+    "edit":               [ "{{ entry['rf_pk'] }}", "{{ entry['marydone'] }}", "{{ entry['rf_shortname'] }}"],
+    "marydone":           "{{ entry['marydone'] }}",
+    "rf_pk":              "{{ entry['rf_pk'] }}",
+    "rf_spdx_compatible": "{{ entry['rf_spdx_compatible'] }}",
+    "rf_shortname":       "{{ entry['rf_shortname'] }}",
+    "rf_fullname":        "{{ entry['rf_fullname']|escape('js') }}",
```

Comment:
You should escape every value, not only some of them.

Reply:
But these are only IDs and Booleans. Do you think they can possibly contain dangerous characters?

Reply:
shortname is a string...